src/Widgets/SnipcartButtonWidget/SnipcartButtonWidgetComponent.js

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import * as Scrivito from 'scrivito';
+import InPlaceEditingPlaceholder from '../../Components/InPlaceEditingPlaceholder';
+
+Scrivito.provideComponent('SnipcartButtonWidget', ({ widget }) => {
+  const pid = widget.get('productId');
+  const name = widget.get('name');
+  const price = widget.get('price');
+  const weight = widget.get('weight');
+  const description = widget.get('description');
+
+  if (!pid) {
+    return (
+      <InPlaceEditingPlaceholder>
+        Provide a product details in the widget properties.
+      </InPlaceEditingPlaceholder>
+    );
+  }
+
+  return (
+    <Scrivito.ContentTag 
```

Comment:
Are button and in-place-editing compatible? If not, we need to do something about it.